modules/Vtiger/helpers/BrowsingHistory.php

```diff
@@ -66,17 +66,28 @@ public static function saveHistory($title)
 		if (empty(App\User::getCurrentUserId())) {
 			return false;
 		}
+		
 		$url = App\RequestUtil::getBrowserInfo()->requestUri;
-		if (empty($url)) {
-			$url = '/';
+		parse_str( parse_url( $url, PHP_URL_QUERY), $url_array );
```

Comment:
$url_array - must be declared in CamelCase [http://pl.wikipedia.org/wiki/CamelCase].
N4 
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files